doc/source/user_guide/io.rst

```diff
@@ -3453,7 +3453,8 @@ Excel files
 The :func:`~pandas.read_excel` method can read Excel 2007+ (``.xlsx``) files
 using the ``openpyxl`` Python module. Excel 2003 (``.xls``) files
 can be read using ``xlrd``. Binary Excel (``.xlsb``)
-files can be read using ``pyxlsb``.
+files can be read using ``pyxlsb``. Also, all this formats can be read using ``python-calamine``,
+but this library has some different behavior from other libraries (mostly for ``.ods``).
```

Comment:
Is there documentation that gives more details on this?

Reply:
Below this, we document which function is used by the various engines so users can determine which `engine_kwargs` they pass. Can you add a line about Calamine there.

Reply:
1) I think, now differences are not too large for explain it in documentation and I removed it:
* calamine can return timedelta;
* some difference from odfpy (mostly OdsReader), such as calamine follows reference and extract value from xml attributes, not from text (time and boolean, for example).
2) added.

I added small chapter for Calamine.